src/components/ProfileForm/components/ProfileFormStep0.tsx

```diff
@@ -128,37 +129,27 @@ const ProfileFormStep0 = ({
         <ProfileFormLabelledInput
           label="Create a skincare routine"
           name="interests"
-          value="routine"
+          value={ProfileGoals.CREATE}
           type="checkbox"
-          checked={localData.goals.includes("create")}
-          onChange={() => onGoalChange("create")}
+          checked={localData.goals.includes(ProfileGoals.CREATE)}
+          onChange={() => onGoalChange(ProfileGoals.CREATE)}
         />
         <ProfileFormLabelledInput
           label="Improve my skincare routine"
           name="interests"
-          value="improve"
+          value={ProfileGoals.IMPROVE}
           type="checkbox"
-          checked={localData.goals.includes("improve")}
-          onChange={() => onGoalChange("improve")}
+          checked={localData.goals.includes(ProfileGoals.IMPROVE)}
+          onChange={() => onGoalChange(ProfileGoals.IMPROVE)}
         />
         <ProfileFormLabelledInput
           label="Discover new products"
           name="interests"
-          value="new_products"
+          value={ProfileGoals.DISCOVER}
           type="checkbox"
-          checked={localData.goals.includes("discover")}
-          onChange={() => onGoalChange("discover")}
+          checked={localData.goals.includes(ProfileGoals.DISCOVER)}
+          onChange={() => onGoalChange(ProfileGoals.DISCOVER)}
         />
-        <ProfileFormLabelledInput
-          label="Other"
-          name="interests"
-          value="other"
-          type="checkbox"
-          checked={localData.goals.includes("other")}
-          onChange={() => onGoalChange("other")}
-        >
-          <input type="text" className="border" />
-        </ProfileFormLabelledInput>
       </fieldset>
```

Comment:
A lot of duplciate code here.

We have the same `<ProfileFormLabelledInput>` component multiple times. Use the same technique from the other PR:

> Create an Array of items, and map over them. Thus, we won't have this component so many times in the UI,